LoadingScrollView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  LoadingScrollView.swift
+//  Uplift
+//
+//  Created by Cameron Hamidi on 2/19/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+class LoadingScrollView: UIView {
+
+    static let defaultLoadingArrangement: [LoadingCollectionViewType] = [.smallGrid, .horizontallyScrolling, .largeGrid]
```

Comment:
why are we making this `static`?

Reply:
It's used as a default argument for the initializer, and the array is too long to put in the function header.